.evergreen/config.yml

```diff
@@ -144,7 +145,7 @@ functions:
             # If this was a patch build, doing a fresh clone would not actually test the patch
             cp -R ${PROJECT_DIRECTORY}/ $DRIVERS_TOOLS
           else
-            git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
+            git clone --branch GODRIVER-2607 https://github.com/blink1073/drivers-evergreen-tools.git $DRIVERS_TOOLS
```

Comment:
Shall we wait for the merging of https://github.com/mongodb-labs/drivers-evergreen-tools/pull/365?